### PR TITLE
Fix invalid specifier in `stone` requirement

### PR DESCRIPTION
**Pull request by sagarvora**
As mentioned in https://github.com/pypa/pip-audit/issues/445#issuecomment-1363101453:

> [PEP 440](https://peps.python.org/pep-0440/) doesn't say it directly, but the language implies that the .* is only valid on exact comparison operators (e.g. == and !=), nor ordered comparisons (e.g. >=).
> 
> In particular, a comparison like >=3.5.* is redundant: it has the exact same meaning as >=3.5.

This is causing `pip-audit` to fail, like so:
https://github.com/frappe/frappe/actions/runs/3811860090/jobs/6484759800#step:4:183

## **Checklist**

**General Contributing**
- [x] Have you read the Code of Conduct and signed the [CLA](https://opensource.dropbox.com/cla/)?

**Is This a Code Change?**
- [x] Non-code related change (markdown/git settings etc)
- [ ] SDK Code Change
- [ ] Example/Test Code Change

**Validation**
- [x] Does `tox` pass?
- [x] Do the tests pass?

### CODE REVIEW

**Comment by greg-db** [comment]
Thanks for sharing this! I'll ask the team to review this.

**Comment by barredterra** [comment]
I just spent an hour trying to solve this problem. But of course @sagarvora is weeks ahead 😁

**Comment by sagarvora** [comment]
@greg-db Any update?

**Comment by greg-db** [comment]
@sagarvora This is still open with the team, but I don't have an update on it yet.

**Review by sderickson — APPROVED** [review]
LGTM! Sorry for the delay.

**Review by sderickson — APPROVED** [review]

**Comment by sderickson** [comment]
Hmm, the checks aren't running for this PR for some reason. Might try adding another commit just to get things going. Will pick at it also.

**Comment by sagarvora** [comment]
Hi @sderickson,

Thanks for approving the PR. I tried following:
- changed the commit message slightly & did a force-push
- signed the CLA (again), I had earlier filled the form manually instead of logging in with Github

Currently, the checks seems to be requiring approval from a maintainer.

**Comment by sagarvora** [comment]
@sderickson Can we release this please? It should fix #458 and close #464.

**Comment by sagarvora** [comment]
@sderickson The release `v11.36.1` doesn't show on [PyPI](https://pypi.org/project/dropbox/#history). Did you miss a step?

**Comment by greg-db** [comment]
@sagarvora Thanks for the note. That release has not been successfully added on PyPI. We'll follow up once we have an update.

**Comment by ghane** [comment]
Hi, could you please push this to PyPi?  I am trying to install a downstream dependant, and this is making that uninstallable.

Thank you

**Comment by sagarvora** [comment]
@ghane This has already veen released  as of [v11.36.1](https://github.com/dropbox/dropbox-sdk-python/releases/tag/v11.36.1)

**Comment by greg-db** [comment]
@ghane Thanks for the note! This is open with the team. I'll follow up here once that's published to PyPI.

**Comment by greg-db** [comment]
v11.36.2 is now published on PyPI.

**Pull request merged**
